src/telnet/mod.rs

```diff
@@ -166,7 +166,9 @@ async fn run_processor<R: AsyncRead + Unpin, H: Handler>(
     loop {
         match read_half.read(&mut buf).await {
             Ok(n) => {
-                (handler, remote) = processor.process(&buf[..n], handler, remote).await?;
+                let (_handler, _remote) = processor.process(&buf[..n], handler, remote).await?;
```

Comment:
Perhaps name them `handler_`. Variables starting with (`_`) can bypass unused warnings. Even when these are not used, it triggers other compile errors. I think it's better not to do this when bypassing unused warnings is not the intention here.

Reply:
Oops. I was not aware of that language feature. Fixed.